CHANGELOG.rst

```diff
@@ -62,6 +62,9 @@ Changed
   trackers are still loaded from pickle but will be dumped as json in any subsequent
   save operations.
 - Event brokers are now also passed to custom tracker stores (using the ``event_broker`` parameter)
+- Print info message when running Rasa X and and custom model server url was specified in ``endpoints.yml``
```

Comment:
these lines are in the wrong part of the changelog. Please make sure they end up in the `unreleased 1.5` part

Reply:
should be in the proper location now